cwm-data/src/main/java/org/cast/cwm/data/Period.java

```diff
@@ -71,7 +71,13 @@ public class Period extends PersistedObject implements Comparable<Period> {
 	 */
 	@Column(unique = true, nullable = false)
 	private String classId;
-	
+
+	/**
+	 * Identifier for LTI reference.
+	 */
+	@Column(unique = true)
+	protected String ltiId;
```

Comment:
My understanding is that these LTI IDs are unique within each LMS that provides them to us (that is, within the Site). But they may not be unique globally for us.

Reply:
I decided to handle this during LTI launch: all "ltiId"s are prefixed with the site "guid", e.g. here "3662"

`3662-8ece48906aab56e5ba18`